third_party/proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -0,0 +1,137 @@
+syntax = "proto3";
```

Comment:
for the third party protos -- would it be possible with a submodule or a build script that will be pinned against the specific version-tag/commit sha on the external repositories?

Reply:
We have to create submodule for the whole cosmos-sdk, can't have only the subdirectory.

Reply:
that's ok -- the integration gRPC tests also cloned the whole repos

Reply:
I removed the unneeded third party protos now, we actually only need the `pagination.proto`